internal/backends/qt/qt_widgets/tablecolumn.rs

```diff
@@ -0,0 +1,167 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+use i_slint_core::{input::FocusEventResult, items::SortOrder};
+
+use super::*;
+
+#[repr(C)]
+#[derive(FieldOffsets, Default, SlintElement)]
+#[pin]
+pub struct NativeTableColumn {
```

Comment:
This is not really a coluln, it is a header section.   "NativeTableHeaderSection" seem like a better name.

Reply:
Describes the struct  better will change that, thank you for the feedback.